.github/workflows/build.yml

```diff
@@ -0,0 +1,36 @@
+name: Build and run
+on:
+  push:
+    branches:
+      - master
+      - develop
+  pull_request:
+    branches:
+      - master
+      - develop
+jobs:
+  build-gfortran:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Setup preCICE
+        uses: precice/setup-precice-action@main
```

Comment:
Unless the extremely latest (<1 day) preCICE development state is required for testing, using the setup-precice-action is an overkill as it will take a lot more time than using the [nightly](https://hub.docker.com/layers/precice/precice/nightly/images/sha256-29ef2a1143f7168982fb42dc4766a14223ff5dc78505080063d6bfe67882e503?context=explore) image.